utilities/svgs.js

```diff
@@ -113,13 +120,35 @@ export const svgs = {
     title: 'Location',
   },
 
+  minus: {
```

Comment:
@kburnell added svgs for plus, minus and preview. Note: they are highly untested, use at your own risk! 😛Lemme know what else you need.